lib/message.rb

```diff
@@ -1,7 +1,9 @@
+require 'pg'
+
 class Message 
   def self.all 
-    [
-      'This is a peep!'
-    ]
+    connection = PG.connect(dbname: 'chitter')
+    result = connection.exec('SELECT * FROM peeps;')
+    result.map { |messages| messages['message']}
```

Comment:
cool use of map!

it's a little unclear what this is saying - but I'm guilty too!!